Terminal.Gui/Input/KeyBinding.cs

```diff
@@ -97,7 +97,7 @@ public void Add (Key key, KeyBindingScope scope, params Command [] commands)
             throw new ArgumentException (@"At least one command must be specified", nameof (commands));
         }
 
-        if ((key == null) || !key.IsValid)
+        if (key == null || !key.IsValid)
```

Comment:
Another case of improper null check but also place to use an object pattern match, avoiding the binary or.